portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -172,6 +179,12 @@ private int getNumberOfCommentsToDisplay(HttpServletRequest request) {
       return numberOfComments;
     }
 
+    /* Returns comments for a certain tag */
+    private String getWhichTagToDisplay(HttpServletRequest request) {
+      String tagToDisplay = getParameter(request, "comments-tag", "");
+      return tagToDisplay;
```

Comment:
Several things:
(1) no need to define a local variable if it's just used once and to be returned.
(2) consider just removing this getWhichTagToDisplay() altogether as it's just a one liner
(3) "comments-tag" should a constant variable.